src/kv/RocksDBStore.h

```diff
@@ -159,7 +160,8 @@ class RocksDBStore : public KeyValueDB {
     compact_thread(this),
     compact_on_mount(false),
     disableWAL(false),
-    enable_rmrange(cct->_conf->rocksdb_enable_rmrange)
+    enable_rmrange(cct->_conf->rocksdb_enable_rmrange),
+    max_items_rmrange(cct->_conf->rocksdb_max_items_rmrange)
```

Comment:
i don't think this would compile. for accessing the new-style options, you need to put
```c++
max_items_rmrange(cct->_conf->get_val<uint64_t>("rocksdb_max_items_rmrange"))
```

Reply:
thanks again, updated

Reply:
sorry, should be

`_conf.get_val<uint64_t>(...)`.